feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md

```diff
@@ -122,17 +122,35 @@ It is foreseen that implementation may drop ISIS adjacency if all members of LAG
 are set with forwarding-viable = FALSE. This scenario may be
 handled via the yet to be defined deviation `logicalInterfaceUPonNonViableAll`.
```

Comment:
This is outdated; the readme requires an update to match the latest spec (https://github.com/openconfig/public/blob/a2c7a09e619c56782b3f4495a065fd7c34ef1175/release/models/interfaces/openconfig-if-sdn-ext.yang#L69)

cc @rszarecki @sachendras @dplore

Reply:
@rszarecki @dplore Looking for your comments. Please let me know if any addition is required in this test.

Reply:
Updated as Suggested